query-utils.js

```diff
@@ -121,10 +121,38 @@ export function getRelatedDecisionType( decisionType, hasCKB ) {
 
 }
 
-export function prepareQuery( { fromEenheid, forEenheid, ckbUri, decisionTypeData, forDecision }) {
+export function ckbDecisionTypeToRelatedType(decisionType) {
+  // See: https://docs.google.com/spreadsheets/d/1DIQkJTCy3Z16xsZaE2QFOhXCQBDKS4ifeYV_592DxRw/edit?gid=1195904946#gid=1195904946
+
+  const ckbDecisionTypes = {
+    // Advies bij jaarrekening eredienstbestuur
+    "https://data.vlaanderen.be/id/concept/BesluitDocumentType/672bf096-dccd-40af-ab60-bd7de15cc461":
+      // Jaarrekening.
+      "https://data.vlaanderen.be/id/concept/BesluitType/e44c535d-4339-4d15-bdbf-d4be6046de2c",
+
+    // Budgetten(wijzigingen) - Indiening bij representatief orgaan
+    "https://data.vlaanderen.be/id/concept/BesluitDocumentType/18833df2-8c9e-4edd-87fd-b5c252337349":
+      // Budget(wijziging) - Indiening bij centraal bestuur of representatief orgaan
+      "https://data.vlaanderen.be/doc/concept/BesluitType/d463b6d1-c207-4c1a-8c08-f2c7dd1fa53b",
+
+    // Budgetten(wijzigingen) - Indiening bij toezichthoudende gemeente of provincie
+    "https://data.vlaanderen.be/id/concept/BesluitDocumentType/ce569d3d-25ff-4ce9-a194-e77113597e29":
+      // Budget(wijziging) - Indiening bij centraal bestuur of representatief orgaan
+      "https://data.vlaanderen.be/doc/concept/BesluitType/d463b6d1-c207-4c1a-8c08-f2c7dd1fa53b",
```

Comment:
Something is not right with these URI mappings. I created submissions as a EB for both types, but searching for them as a CKB doesn't return any results.